website/main/content/blog/2025-04-21-the-beginning-of-a-spec.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The beginning of a Spec
+date: 2025-04-21
+draft: false
+description: An artifact to foster design discussions and guide the implementation of prototypes.
+tags: [status-update, consensus]
+authors: [adlrocha]
+---
+
+Over the past weeks, my updates have highlighted many of the ideas emerging from our open design
+discussions. Now that we have a clearer direction for the design, I wanted to consolidate these
+ideas into a draft spec. This will serve as a foundation for implementing the first few prototypes,
+while also providing a structured way to gather feedback and uncover potential blind spots. I expect
+this spec to suffer significant changes, but it felt like the perfect way to consolidate the ideas,
+get feedback from the community, and unblock Nazar in case he wants to start prototyping some of the
+ideas we've been discussing.
+
+<!--more-->
+
+## Where to find the spec and the first drafts
+
+[PR192](https://github.com/nazar-pc/abundance/pull/192) adds to the repo a new `spec` directory
```

Comment:
There is already `specs` directory (I expect there will be multiple specs for different components): https://github.com/nazar-pc/abundance/tree/main/specs

Reply:
My bad. It didn't exist when I started writing the spec drafts. Fixed the text accordingly.